pkg/controller/jobs/jobset/jobset_controller.go

```diff
@@ -182,6 +184,31 @@ func (j *JobSet) PodsReady() bool {
 	return replicas == readyReplicas
 }
 
+func (j *JobSet) ReclaimablePods() []kueue.ReclaimablePod {
```

Comment:
let's add unit tests (with a test table) for this.

Reply:
done